content/docs/command-reference/repro.md

```diff
@@ -154,6 +155,9 @@ up-to-date and only execute the final stage.
   corresponding pipelines, including the target stages themselves. This option
   has no effect if `targets` are not provided.
 
+- `--pull` - try automatically pulling cached outputs if they are not present in
```

Comment:
okay, a few questions:

1. `try` - what happens if it fails?
2. `pulling` - make it a link to the `dvc pull` probably
3. `cached outputs` - here not sure if it's better to use DVC-tracked outputs. (otherwise when you read it is bit hard mentally since you are they cached but not present in cache).

WDYT? 

@jorgeorpinel ?

Reply:
1. as it was before, it will simply not restore from run-cache.

Addressed 2 and 3. Thank you.

Reply:
Yep, got it. I guess it's fine for now. No reason to further improve this since we don't have run-cache documented anywhere. So we can keep as is -an advanced option.

Reply:
> we don't have run-cache documented anywhere

BTW the run-cache is already mentioned in 6 cmd refs (published) and in the Data Pipelines page of the GS, which I just noticed/realized just now. I thought we were not going to include any info about experiments until it's more stable? Should we remove these mentions or prioritize documenting run-cache? Thanks

Reply:
@jorgeorpinel Let's not remove it. It is in a semi-official state, people already use it due to cml and other sources. We are on our way to cleaning up the ui overall and publishing experiments.

Reply:
> Let's not remove remove it. It is in a semi-official state

OK

> Run-cache doc by itself doesn't really mean anything to the users

Much disagree 🐶 I mean it's not so important whether it's a stand-alone doc or a new section in existing doc(s), but the basics about run-cache seem like a quite important thing to document to me.

> It only makes sense in particular commands

Yeah anywhere we want to put it as long as it's published would be great since this is already semi-official.

Reply:
@jorgeorpinel Agreed, I've added the run-cache doc ticket to next sprint, just preliminarily. Thanks :slightly_smiling_face:

Reply:
Many thank! 🦊